### PR TITLE
Add item loading

### PR DESCRIPTION
**Pull request by nimkamp**

### CODE REVIEW

File: app/styled-components/Spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import styled, { keyframes } from 'styled-components';
```
Comment (scottPlusPlus) [inline app/styled-components/Spinner.tsx]: there's a "components" folder in /app.   let's stick this in there unless you have objections

**Review by scottPlusPlus — COMMENTED** [review]
Can you add a screen capture of the new spinner?

**Comment by nimkamp** [comment]
https://github.com/scottPlusPlus/uscout/assets/5142680/91a80d53-1adc-4256-ae6e-b6640cfe6685

File: app/styled-components/Spinner.tsx (hunk 1 above)
Comment (nimkamp) [reply app/styled-components/Spinner.tsx]: I moved this. Fixed.

**Pull request merged**